lib/tapioca/gem/pipeline.rb

```diff
@@ -337,7 +337,7 @@ def defined_in_gem?(constant, strict: true)
         return !strict if files.empty?
 
         files.any? do |file|
-          @gem.contains_path?(file)
+          @gem.contains_path?(file.path)
```

Comment:
Hmmm, I am not sure if this will work since `get_file_candidates` probably returns just an enumerable of files. I think it would be better to change `fules_for` to return just filenames instead.

Reply:
Changed it to continue returning a set.